packages/gatsby-theme-carbon/src/components/ArticleCard/article-card.scss

```diff
@@ -1,132 +1,141 @@
-.#{$prefix}--article-card .#{$prefix}--tile {
+// $carbon--white-0
```

Comment:
```suggestion
```

Small clean-up nit, the comments are great while developing but may be better off excluded from the final code.